src/common/components/Header/HeaderScreen.tsx

```diff
@@ -154,7 +154,7 @@ const Header = (props: PropsType) => {
             <Link to='#' className='flex items-center'>
               <img
                 src={addBtn}
-                className='w-20px h-20px mr-2px md:mr-8px'
+                className='w-20px h-20px mr-2px md:mr-2'
```

Comment:
@bulatsalyakhovdev can you change `mr-2` to `mr-2px`

Reply:
@dbizhev tailwind has several default spacing values, `mr-2` is `0.5rem`, which is `8px`. No need to add a `8px` in tailwind.config.js as it's duplicated